packages/client/components/TopBarStartMeetingButton.tsx

```diff
@@ -21,11 +19,12 @@ const MeetingLabel = styled('div')({
 })
 
 const TopBarStartMeetingButton = () => {
+  const location = useLocation()
   const teamId = getTeamIdFromPathname()
   const {history} = useRouter()
 
   const onClick = () => {
-    history.push(`/new-meeting/${teamId}?source=TopBar`)
+    history.push(`/new-meeting/${teamId}?source=TopBar`, {backgroundLocation: location})
```

Comment:
-1 history.replace

Reply:
+1 when teamId is unknown, it feels weird that this url flickers `/source=TopBar` & then goes to `/:teamId`. Nothing breaks, it just feels unpolished to have quick redirects.
 
ideally, the flow would be:
- if teamId is unknown, persist source=TopBar somewhere else (context, localSession, global var, react router location, etc.
- when teamId is known & we're going to redirect to /:teamId, attach the source=TopBar to the end

Reply:
@Dschoordsch can we use `SendClientSegmentEventMutation` instead of `source=TopBar` as a simple fix?

Reply:
Let's just get rid of `source=TopBar` and `source=BottomFab`. I don't think we need this anymore, as it's not a user choice now, but just depends on screen resolution which button is visible.

Reply:
Removed